### DISCUSSION TITLE
Email not generated correctly

### DISCUSSION THREAD

**Discussion by BacicaFlorinTech** [deepgram/community#788] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

I am testing the text to speech api in the playground and I see that it doesn't generate the emails correctly.
I am using this email yoxoma7079@synclane.com. The numbers from the email are not generated and the @ is barely hearable.
Is there a way to reformat the email so it can be generated correctly.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @BacicaFlorinTech, thank you for the feedback, and apologies that the email address isn't being correctly voiced.

I was able to replicate your example, where the numbers are omitted in the output speech. I've reported this internally; it may have to do with some input text cleaning that is removing the numbers before the speech generation. I found that the input text `"yoxoma 7079@synclane.com"` with a space before the numbers was pronounced as expected.

I'll follow up if I do find that this is a generalizable bug that we're able to resolve.

**Reply by BacicaFlorinTech** [reply]
Thank you @jkroll-deepgram! Let us know if any improvements are done.
